src/BuiltinExtensions/ComfyUIBackend/ExtraNodes/SwarmComfyExtra/SwarmYolo.py

```diff
@@ -11,13 +11,17 @@ def INPUT_TYPES(cls):
                 "model_name": (folder_paths.get_filename_list("yolov8"), ),
                 "index": ("INT", { "default": 0, "min": 0, "max": 256, "step": 1 }),
             },
+            "optional": {
+                "sort_order": (["left-right", "top-bottom", "largest-smallest", "default"], ),
```

Comment:
heck is "default"?

Reply:
should I change that to none? Or just add a blurb that says "default - if an index is passed, segment the index, if no index is passed, segment all found masks" or something similar?

Basically default is, just do what this would do if nothing was set. Maybe that's redundant because if not toggled nothing is passed? I'm open to suggestions on proper wording.

Reply:
there is no "default", there's some form of order, always. the code before this PR was explicitly always left-to-right

Reply:
Then I will simply set left-right as the "default" and remove `default` as an option. That will better represent the logic that already exists. Makes things easier that way, and less confusing.